src/ttsim/unit_converters.py

```diff
@@ -0,0 +1,976 @@
+from __future__ import annotations
+
+from typing import overload
+
+TIME_UNIT_LABELS = {
+    "y": "Year",
+    "q": "Quarter",
+    "m": "Month",
+    "w": "Week",
+    "d": "Day",
+}
+
+_Q_PER_Y = 4
+_M_PER_Y = 12
+_W_PER_Y = 365.25 / 7
+_D_PER_Y = 365.25
+
+
+@overload
+def y_to_q(value: int) -> int: ...
+
+
+@overload
+def y_to_q(value: float) -> float: ...
+
+
+def y_to_q(value: float) -> int | float:
+    """
+    Converts values on year level to values on quarter level.
+
+    Parameters
+    ----------
+    value
+        Value on year level to be converted to value on quarter level.
+
+    Returns
+    -------
+    Value on quarter level.
+    """
+    return value * _Q_PER_Y
+
+
+@overload
+def per_y_to_per_q(value: int) -> float: ...
+
+
+@overload
+def per_y_to_per_q(value: float) -> float: ...
+
+
+def per_y_to_per_q(value: float) -> float:
+    """
+    Converts values per year to values per quarter.
```

Comment:
```suggestion
    Convert flows: values per year to values per quarter.
```
Same comment on Paramters/Returns as above. 

And ofc, apply these everywhere.